packages/dev/docs/pages/react-aria/routing.mdx

```diff
@@ -287,8 +287,8 @@ function RootRoute() {
   let router = useRouter();
   return (
     <RouterProvider 
-      navigate={(to, options) => router.navigate({to, ...options})}
-      useHref={to => router.buildLocation({to}).href}>
+      navigate={(href, opts) => router.navigate({...href, ...options})}
```

Comment:
Apologies, had a thought after approving, and I not quite familiar enough, but should this line remain as it was?